marnil-hot/SJ1-PR3-TheDataGenerator
# First commit. Structure and a Simple Hello world

This is the first basic structure. Contains a unit test and a hello world program.
Java 8 is required

**What to do**
To approve this Pull Request:
* At least one person has to perform a code review and once approved add a "+1" in the comments
* At least one person has to test the program. Once approved add a "+2" in the comments

**Setup:**
Download Eclipse (lastest version is currently Neon.2)
Get a GitHub(.com) account. 
Go to https://github.com/marnil-hot/SJ1-PR3-TheDataGenerator/
Add project to Eclipse workspace:
1 )In Eclipse: Windows->Show view->Other->Git->Git Repositories
2) Choose "Clone a Git repository and add to this view"
3) Add the URI "https://github.com/marnil-hot/SJ1-PR3-TheDataGenerator/" and authenticate with yout GitHub username and login
4) Right click on the GIT repository and run "import project"

**How To Test:**
1) Right click on the file "src/main/java/com.sr1.pr3.controller/ConsoleRunner.java" and choose "Run as-> Java Application". It should now open a console with a "Hello World" message
2) Right click on the file "src/main/java/com.sr1.pr3.datagenerator/DataGeneratorTest.java" and choose "Run as-> JUnit Test". A JUnit Window should now appear displaying one passing test case.
